drupal8-example/Makefile

```diff
@@ -0,0 +1,22 @@
+up:
```

Comment:
Why introduce the Makefile in this PR?

There is a Makefile example in another folder of the Git root. That might no be desirable in the long run but is not really related to the NFS stuff(?)

Reply:
Ah, didn't see that one. :) 
It is related to the NFS stuff however (we go from bind volumes to NFS volumes on macOS)

Reply:
Yeah, but having two different Makefile examples around just leads to incompatibility.

Reply:
I agree! I'm removing it. :)